### PR TITLE
Support graphql in backend

### PR DESCRIPTION
**Pull request by livnekes**

### CODE REVIEW

File: backend/api-gw/src/server.ts [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
 
 // TODO - replace this with frontEgg middleware
 app.use('/*', (req, res, next) => {
   req.headers['frontegg-tenant-id'] = '90b02525-0d38-415e-8e23-dc4d8c1c4cb0';
   next();
 });
-app.use('/metrics', proxy(METRICS_URL));
-app.use('/config', proxy(CONFIG_URL));
+
+class AuthenticatedDataSource extends RemoteGraphQLDataSource {
```
Comment (aviadmizrachi) [inline backend/api-gw/src/server.ts:16]: Let's separate it to a different class

File: backend/services/metrics-service/src/db/seed/request.seed.ts [hunk 2]
```diff
@@ -1,40 +1,67 @@
...
+const USER_AGENTS: string[] = [
+  'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36',
+  'Mozilla/5.0 (Linux; U; Android 2.2) AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 Mobile Safari/533.1',
+  'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:80.0) Gecko/20100101 Firefox/80.0',
+  'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36',
+  'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:80.0) Gecko/20100101 Firefox/80.0',
+  'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36 Edg/85.0.564.51',
 ];
+
+const IPS: string[] = ['208.80.235', '185.159.83.104', '208.97.177.47', '45.33.45.6'];
```
Comment (aviadmizrachi) [inline backend/services/metrics-service/src/db/seed/request.seed.ts:17]: 208.80.235 is not a valid IP V4

**Review by aviadmizrachi — CHANGES_REQUESTED** [review]

File: backend/api-gw/src/server.ts (hunk 1 above)
Comment (livnekes) [reply backend/api-gw/src/server.ts]: done

File: backend/services/metrics-service/src/db/seed/request.seed.ts (hunk 2 above)
Comment (livnekes) [reply backend/services/metrics-service/src/db/seed/request.seed.ts]: done

**Review by aviadmizrachi — APPROVED** [review]

**Pull request merged**